public/video-ui/src/components/utils/YouTubeEmbed.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {getStore} from '../../util/storeAccessor';
+
+export function youTubeUrl(id) {
+    return `https://www.youtube.com/watch?v=${id}`;
```

Comment:
The function is supposed to return a link to the YouTube page but was misleadingly in this file. I've just inlined it since it was only called in one place.